clients/fides-js/src/lib/initialize.ts

```diff
@@ -425,8 +426,14 @@ export const initialize = async ({
   }
 
   // eslint-disable-next-line @typescript-eslint/naming-convention
-  const { consent, fides_meta, identity, fides_string, tcf_consent } =
-    fides.cookie;
+  const { fides_meta, identity, fides_string, tcf_consent } = fides.cookie;
+  const consent = applyOverridesToConsent(
+    fides.cookie.consent,
+    fides.experience?.non_applicable_privacy_notices,
+    fides.experience?.privacy_notices,
+    options.fidesConsentFlagType ?? undefined,
+    options.fidesConsentNonApplicableFlagMode ?? undefined,
```

Comment:
Can we add a test for ensuring the GTM related flags get initialized correctly? We can probably just add an assertion to existing tests

Reply:
Those are no longer just for GTM, and the related tests are in the `applyOverridesToConsent` section of `clients/fides-js/__tests__/lib/consent-utils.test.ts`. Is there more you think needs to be done there?

Reply:
Oh, sorry, I think I see what you mean... On the Cypress tests I've added some assertions for checking the window.option values.